include/mega/megaapp.h

```diff
@@ -426,6 +426,11 @@ struct MEGA_API MegaApp
     virtual void dismissbanner_result(error) { }
 
     virtual ~MegaApp() { }
+
+    // External drive notifications
+    bool startDriveMonitor();
+    void stopDriveMonitor();
```

Comment:
MegaApp is just for callbacks - start/stop should be invoked by client->mDriveInfoCollector.start()/stop()

Reply:
But... it's not, whether there are functions for that or direct access to a member, and IHMO the former is cleaner.


> DriveInfoCollector and this IDriveNotify can be the same class

(interestingly, I cannot reply to that comment directly)
Yup, you're right. Earlier, it seemed to make more sense like it currently is, but not so much anymore. I'll quickly update it.

Reply:
You can wrap them if you want, but MegaApp is only for callbacks.
If it feels cleaner, you can prepare something like:
``` cpp
void MegaClient::startDriveMonitor()
{
    mDriveInfoCollector.start();
}
```

Reply:
That's more or less what is already implemented. So IIUC, no change for this one, right?

Reply:
Every function declared in MegaApp is a callback to be overridden by the client, and called by the SDK core when the relevant event has occurred.   These two new functions break that design, so please move them elsewhere - either Sergio's suggestion or mine or something else is ok, but MegaApp is designed to be only for callbacks.    thanks

Reply:
Actually I think in the current version, these two function declarations are unused anyway?  They don't seem to be called or have a definition - it's the ones in MegaClient that are used - you can just remove those two lines.  thanks

Reply:
They had definitions, and they were used from `MegaApi` and `megacli`.
Removed now.